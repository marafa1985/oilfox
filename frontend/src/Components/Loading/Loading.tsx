import React from "react"
import './Loading.scss'


const Loading: React.FC = () => {
    return (
        <div className="loading">
            <div className="spinner">
                <div className="double-bounce1" />
                <div className="double-bounce2" />
            </div>
        </div>
    )
}

export default Loading;