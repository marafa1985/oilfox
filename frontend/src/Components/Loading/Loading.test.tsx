import React from "react";
import { create } from "react-test-renderer";
import Loading from './Loading'
describe("Loading component", () => {
    it("use jest snapshot since it will not changed", () => {
        const button = create(<Loading />);
        expect(button.toJSON()).toMatchSnapshot();
    });
}
);
