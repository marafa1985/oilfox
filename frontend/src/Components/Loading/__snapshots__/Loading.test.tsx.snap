// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component use jest snapshot since it will not changed 1`] = `
<div
  className="loading"
>
  <div
    className="spinner"
  >
    <div
      className="double-bounce1"
    />
    <div
      className="double-bounce2"
    />
  </div>
</div>
`;
